.github/workflows/pr-workflow.yml

```diff
@@ -1,7 +1,7 @@
-name: Deploy workflow
```

Comment:
The idea was to have one workflow to run on PRs(set simply as push for now, as it would not make much difference for the time being) and one to run on main to deploy the app. The purpose for this workflow was to run test and deploy the app, thus it should run on main, not on PR.
Shouldn't we apply these changes on the other workflow instead?

Reply:
Absolutely, they can be merged in a single workflow (since now they are the same).

But in this case I'd just purge the Deploy workflow.
Currently, the Deploy workflow is being executed on each push on main, which is not a deploy.

I'd imagine the Deploy workflow to be executed only under specific conditions (e.g. tag, filter on files, etc..).

Reply:
It depends: I publish to Play Store on my pet projects at every push to `main`.
Maybe we can remove the Deploy Workflow for now and just leave the PR one, till we decide how and when we want to deploy

Reply:
Oh I got what you meant.
I'd remove the Deploy Workflow for now for the following two reasons:
1. We probably won't have nothing to deploy for quite some time so maintaining the "same" workflow for the moment isn't ideal. We can just copy it whenever we decide to add a Deploy workflow
2. Personally I'd never deploy on every push on `main`, but rather doing that in a cron job for nightly builds or on tags for production builds. This doesn't mean that the strategy I prefer values less or more than the one you prefer bur rather that we probably need some discussione about it

What do you think?

Reply:
Completely agree! My method works on my pet, as I commit directly to main, then I push once and it gets deployed. This probably would not fit an environment where we have many PR that got frequently merged to main

Reply:
Fixed!